src/nvidia-cuda/install.sh

```diff
@@ -33,6 +36,11 @@ check_packages() {
     fi
 }
 
+if [ $VERSION_CODENAME = `bookworm` || $VERSION_CODENAME = `jammy`] && [ $CUDA_VERSION < 11.7 ]; then  
```

Comment:
Can you fix https://github.com/devcontainers/features/actions/runs/8666785264/job/23768360746?pr=942#step:4:125

Reply:
done

Reply:
It's still failing https://github.com/devcontainers/features/actions/runs/8667039013/job/23769166488?pr=942#step:4:125

Reply:
done